jest.config.js

```diff
@@ -6,5 +6,6 @@ module.exports = {
   },
   testEnvironment: 'jest-environment-jsdom',
   testMatch: ['<rootDir>/tests/unit/**/*.spec.{js,jsx,ts,tsx}'],
-  setupFilesAfterEnv: ['<rootDir>/src/setupTests.ts']
+  setupFilesAfterEnv: ['<rootDir>/src/setupTests.ts'],
+  transformIgnorePatterns: ['node_modules/(?!(msw)/)']
```

Comment:
Most of the time, you don't want to transform your node_modules because it can be slow and libraries should be shipped in a way that they are already ready for usage in most environments. Here, node_modules are ignored from transformations, the msw module should still be transformed.